### PR TITLE
Progress update update

### PR DESCRIPTION
**Pull request by glitterbutts**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,13 +7,14 @@ This repo shows the current status of the symbols found for the dbrand Complete
 ## Current solve status
 Still currently unsolved. Cards that have been collected so far can be seen [here](/acquiredSymbols).
 
-As of May 17, 2021, dbrand has "confirmed" that a total of 23 cards have been acknowledged, you can see the message [here](https://discord.com/channels/520021794380447745/832309320934621234/837808784562716763) (If you're in the Discord), or below.
+As of May 17, 2021, dbrand has "confirmed" that all 23 cards have been acquired, you can see the message in the pinned section of the CTC2 discord channel.
```
Comment (tooboredtocode) [inline README.md]: This just makes it less helpful for people not in the discord

File: README.md [hunk 2]
```diff
@@ -7,13 +7,14 @@ This repo shows the current status of the symbols found for the dbrand Complete
...
 Additionally they have confirmed that we have everything needed, to solve this challange.
 
-Given the clue `symbols < cards` we've concluded that we are looking for 3 words made up of 21 characters and two other unknowns.
-While we initially assumed those to be spaces, recent messages led us to conclude otherwise.
+Given the clue `symbols < cards`, we've concluded that we are looking for a phrase made up of 21 characters and two other unknowns.
+We believe these two unknowns are duplicate symbols. Using our brains, we've deduced that the two duplicate characters can be `C` `D` `H` or `M`.
+robot069 has "confirmed" this theory as of May 25, 2021.
 
-![Missing confirmation](message.png)
+![Confirmation](message.png)
```
Comment (tooboredtocode) [inline README.md:17]: Why delete the old file, thats still important?

**Review by tooboredtocode — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Glibbl) [reply README.md:10]: Fuck the people not in the Discord

File: README.md (hunk 2 above)
Comment (glitterbutts) [reply README.md:17]: Old file was the screenshot of dbrand saying that all 23 cards had been acquired. New one is 069 reaffirming the mystery behind the two extra cards. idk, unverified people in the server can see the channel and look at the pins, but they can't post their own messages.

File: README.md (hunk 1 above)
Comment (glitterbutts) [reply README.md]: *They should probably join the discord.* Link is above.

**Review by tooboredtocode — APPROVED** [review]
lgtm

**Pull request merged**